### ISSUE TITLE
Training set is smaller than validation set

### ISSUE THREAD

**Issue by D3nz13** [Piotr45/cmos-preprocessor#25]
Proportion of test set in one split is reversed resulting in the validation set being bigger than training set
